SCons/Tool/install.py

```diff
@@ -127,9 +172,11 @@ def copyFunc(dest, source, env):
 #
 def copyFuncVersionedLib(dest, source, env):
     """Install a versioned library into a destination by copying,
-    (including copying permission/mode bits) and then creating
-    required symlinks."""
 
+    Mode bits will be copied as well.
```

Comment:
please leave "permission" in this comment.
Some people may not understand mode bits..

Reply:
OK.  I used the terminology from `shutil.copymode`, but there's no reason to be a bit more descriptive.